### PR TITLE
Visual bug fixes related to UHF for website

### PR DESCRIPTION
**Pull request by lynamemi**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [X] Include a change request file using `$ npm run change`

#### Description of changes

This addresses bugs filed internally for the current website dogfood environment:
- Extended angle on Blog and Resources pages to fit new app width
- Added more specific link selectors to target link colors on the Homepage
- Component page header is now sticking to the top, no more gap leftover from previous header

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by lynamemi** [comment]
@Jahnp, John thought it best to merge this one into master - it's the same as the one you reviewed yesterday, just a tweak to some padding variables https://github.com/OfficeDev/office-ui-fabric-react/pull/2874

File: apps/fabric-website/src/components/App/App.scss [hunk 1]
```diff
@@ -40,10 +33,17 @@
...
-      border-bottom: 1px solid WindowText;
+      border-right: 1px solid $ms-color-white;
+      border-left: 1px solid $ms-color-white;
+      border-bottom: 1px solid $ms-color-white;
+    }
+
+    @include high-contrast-black-on-white  {
+      border-right: 1px solid $ms-color-black;
+      border-left: 1px solid $ms-color-black;
+      border-bottom: 1px solid $ms-color-black;
```
Comment (kristoferbrown) [inline apps/fabric-website/src/components/App/App.scss]: Why did this get changed back from using the `WindowText` system color? Using the system color allows it to support any high-contrst palette with a single media query.

Comment (lynamemi) [reply apps/fabric-website/src/components/App/App.scss]: This is likely due to the crazy branch switching we've been doing - I'll look into it!

Comment (lynamemi) [reply apps/fabric-website/src/components/App/App.scss]: Yeah, @kristoferbrown, I think it was the former - I'll switch it back. Thanks for paying attention!

**Pull request merged**